cdap-cli/src/main/java/co/cask/cdap/cli/command/security/AddRoleToPrincipalCommand.java

```diff
@@ -47,13 +47,12 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String principalName = arguments.get("principal-name");
     client.addRoleToPrincipal(new Role(roleName), new Principal(principalName, Principal.PrincipalType.valueOf
       (principalType.toUpperCase())));
-    output.printf("Successfully added role '%s' to %s %s\n", roleName, principalType, principalName);
+    output.printf("Successfully added role '%s' to '%s' '%s'\n", roleName, principalType, principalName);
   }
 
   @Override
   public String getPattern() {
-    return String.format("add role <%s> to %s <%s> %s <%s>", ArgumentName.ROLE_NAME,
-                         ArgumentName.PRINCIPAL_TYPE, ArgumentName.PRINCIPAL_TYPE, ArgumentName.PRINCIPAL_NAME,
+    return String.format("add role <%s> to <%s> <%s>", ArgumentName.ROLE_NAME, ArgumentName.PRINCIPAL_TYPE,
```

Comment:
I took care of it in this PR as it seemed to be related issue. I did it in InMemAuthorizer but I can move it to AuthHandler but I don't think we should handler it in Handler just because we don't want the authbackends to do the same. 

For example our InMemAuthorizer allow granting/revoking permissions on user/group or to roles. When list privileges is called it gets the privileges for the principal and if its not a role then it gets privileges given to all the roles it belongs too. same for enfore. Not sentry will not do this because there  privileges only belong to role. So, I think we should not do  such implementation checks in handler. thoughts ?